foreman/foreman.spec

```diff
@@ -656,7 +678,7 @@ exit 0
 # always attempt to reencrypt after update in case new fields can be encrypted
 %{foreman_rake} db:migrate db:compute_resources:encrypt >> %{_localstatedir}/log/%{name}/db_migrate.log 2>&1 || :
 %{foreman_rake} db:seed >> %{_localstatedir}/log/%{name}/db_seed.log 2>&1 || :
-%{foreman_rake} apipie:cache >> %{_localstatedir}/log/%{name}/apipie_cache.log 2>&1 || :
+%{foreman_rake} apipie:cache:index >> %{_localstatedir}/log/%{name}/apipie_cache.log 2>&1 || :
```

Comment:
I'm curious what happens when foreman and 3 plugins all with same :index in posttrans are installed. Will it trigger the :index four times?